kubernetes/loculus/values.yaml

```diff
@@ -96,6 +96,16 @@ defaultOrganismConfig: &defaultOrganismConfig
         noInput: true
         perSegment: true
         oneHeader: true
+      - name: geoLocLatitude
+        displayName: Latitude
+        header: Sample details
+        type: float
+        guidance: Geo-coordinate latitude in decimal degree (WGS84) format, i.e. values in range -90 to 90.
+      - name: geoLocLongitude
+        displayName: Longitude
+        header: Sample details
+        type: float
+        guidance: Geo-coordinate longitude in decimal degree (WGS84) format, i.e. values in range -180 to 180.
```

Comment:
Maybe just for sanity checks, we could add that West of Greenwich is Negative, so US is negative, Germany positive